pkg/oauth/manager/manager.go

```diff
@@ -42,13 +42,22 @@ type AuthorizeGenerateRequest struct {
 	Request      *http.Request
 }
 
-type AccessTokenGenerateRequest struct {
+type OauthTokensRequest struct {
 	ClientID     string
 	ClientSecret string
 	Code         string // authorization code
+	RefreshToken string // refresh token
```

Comment:
Where to find the expiration of RefreshToken?

Reply:
it can be configurated in horizon config file